app/bundles/LeadBundle/Views/Company/company.html.twig

```diff
@@ -0,0 +1,229 @@
+{#
+  Variables
+    - company
+    - fields
+    - items
+    - permissions
+    - engagementData
+    - security
+    - page
+    - totalItems
+    - limit
+#}
+{% extends 'MauticCoreBundle:Default:content.html.twig' %}
+
+{% block headerTitle %}{{ company.name|purify }}{% endblock %}
```

Comment:
Do we need to purify that many fields? It's needed only for fields that can have HTML tags. I don't expect company name or a field label should have a HTML tags so I'd leave it without any filter. Purifying is quite expensive operation.

Reply:
I just used the php file as a template and copied over those. I'll go over it and remove a lot of those.

Reply:
Thanks! The PHP templates were using `$this->escape()` to avoid XSS attacks. That's how Twig treats all values by default. Only for the values that need to output some HTML tags like description fields should use purify to avoid XSS attacks while rendering HTML.